core/src/main/java/org/apache/calcite/sql/fun/SqlLibrary.java

```diff
@@ -85,7 +85,10 @@ public enum SqlLibrary {
   SNOWFLAKE("f", "snowflake"),
   /** A collection of operators that are in Apache Spark but not in standard
    * SQL. */
-  SPARK("s", "spark");
+  SPARK("s", "spark"),
+  /** A collection of operators that are in Apache Druid but not in standard
+   * SQL. */
+  DRUID("d", "druid");
```

Comment:
We shouldn't add DRUID as a SqlLibrary. DRUID is not a Jdbc database. Each adapter has a specific section that indicates which functions are supported and which are not by the current adapter. This is because only the supported functions can enable some API conversions.